spec/lib/granite/action/preconditions/object_precondition_spec.rb

```diff
@@ -0,0 +1,56 @@
+RSpec.describe Granite::Action::Preconditions::ObjectPrecondition do
+  before do
+    stub_class(:title_precondition, Granite::Action::Precondition) do
+      def call(expected_title:, **)
+        decline_with(:wrong_title) if title != expected_title
```

Comment:
Hey @dziulius 
I think there will be one issue with locales. 
Now if you want to define a locale for the `:wrong_title` you will need to define it for the business action where `ObjectPrecondition` is used, like this:
```
granite_action:
    errors:
      models:
        ba/some_action:
          attributes:
            base:
              wrong_title: 'The title is wrong'
```

It would be great to make it work for the `object_precondition` namespace, like this:
```
  granite_action:
      object_precondition:
            wrong_title: 'The title is wrong'
```
So that we won't need to duplicate locales if they are used in multiple BAs. 
WDYT?

Reply:
Good point, though I think we should keep option to overwrite it per action basis, so lookup would be something like this: `action, parent_action, precondition`.

Reply:
I can't find a nice way to do this, without rewriting how `decline_with` works.

Reply:
sounds good to me